osu.Game/Rulesets/Difficulty/Skills/StrainSkill.cs

```diff
@@ -72,19 +72,21 @@ private void saveCurrentPeak()
         /// Sets the initial strain level for a new section.
         /// </summary>
         /// <param name="time">The beginning of the new section in milliseconds.</param>
-        private void startNewSectionFrom(double time)
+        /// <param name="current"></param>
```

Comment:
Added parameter xmldocs here need filling. Something as simple as `The difficulty hit object to calculate the initial strain for.`